intake_thredds/source.py

```diff
@@ -69,6 +69,12 @@ def __init__(
         self.urlpath = url
         if 'simplecache::' in url:
             self.metadata.update({'fsspec_pre_url': 'simplecache::'})
+        if isinstance(path, str):
+            path = list(path)
```

Comment:
Calling `list()` on a string will result in a list of characters.

```suggestion
            path =[path]
```

Reply:
ah. now I get it. thanks